src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -89,7 +89,7 @@ export class AccountsExternalComponent implements OnInit, OnDestroy {
       name: 'ORCID',
       source: TokenSource.ORCID,
       bold: '',
-      message: 'ORCID credentials are used for linking ORCID IDs to workflows.',
+      message: 'ORCID credentials are used for linking ORCID IDs to workflows published on Zenodo.',
```

Comment:
They'll be used for more than that, as we discussed at storytime, but I don't know how comprehensive you want to be. If you can think of something more general better that would be good, but don't need to block the PR for it.

Reply:
Maybe `ORCID credentials are used for vouching for workflows and tools plus linking ORCID IDs to workflows published on Zenodo.`